comiccrawler/core/__init__.py

```diff
@@ -672,3 +680,10 @@ def analyze_info(mission, mod):
 	remove_duplicate_episode(mission)
 
 	print("Analyzing success!")
+
+def format_number(title, format):
+	"""第3卷 --> 第003卷"""
+	def replacer(match):
+		number = match.group()
+		return format.format(int(number))
+	return re.sub("\d+", replacer, title)
```

Comment:
Maybe we should try to trim the mission title from the episode title first?

Some title examples:
```
泰安路47号 第006回
http://www.dm5.com/manhua-taianlu-47-hao/

原作版108(7)
第28-29話
http://tw.ikanman.com/comic/9637/
```